lib/api/application.go

```diff
@@ -134,6 +95,21 @@ func SetupApplication(app types.Application) types.ResponseError {
 		}
 	}
 
+	_, err = docker.ExecProcess(app.GetContainerID(), []string{"git", "init"})
+	if err != nil {
+		return types.NewResErr(500, "git init unsuccessful", err)
```

Comment:
```suggestion
		setup <- types.NewResErr(500, "git init unsuccessful", err)
		return
```
This should be like this

Reply:
[here](https://github.com/sdslabs/gasper/blob/44c8d3adcd6bb84698a20e3d3fbe388a54402669/lib/api/application.go#L121)

This is the function definition (as on the develop branch).
`setup channel` was not passed as a parameter in `SetupApplication()` function, I did not alter this. Will I still have to make this change?